content/docs/02-foundations/02-providers-and-models.mdx

```diff
@@ -31,6 +31,7 @@ The AI SDK comes with several providers that you can use to interact with differ
 - [Google Generative AI Provider](/providers/ai-sdk-providers/google-generative-ai) (`@ai-sdk/google`)
 - [Google Vertex Provider](/providers/ai-sdk-providers/google-vertex) (`@ai-sdk/google-vertex`)
 - [Mistral Provider](/providers/ai-sdk-providers/mistral) (`@ai-sdk/mistral`)
+- [Grok](/providers/ai-sdk-providers/grok) (`@ai-sdk/grok`)
```

Comment:
Is grok just their model name, like claude? if so we should name the provider `xai` - they probably will have more models in the future